src/AlgebraicGeometry/Schemes/Covering/Objects/Attributes.jl

```diff
@@ -46,3 +46,30 @@ function glueing_graph(C::Covering)
   return C.glueing_graph
 end
 
+@doc raw"""
+    decomposition_info(C::Covering)
+
+Return an `IdDict` `D` with the `patches` of `C` as keys and values `D[U]` a list 
+of elements ``f₁,…,fᵣ ∈ 𝒪(U)``. These elements are chosen so that for 
+``Xᵤ = V(f₁,…,fᵣ) ⊂ U`` the scheme ``X`` covered by ``C`` decomposes as a 
+disjoint union ``X = ∪ Xᵤ``.
```

Comment:
This should explain the idea.

Reply:
I don't get it.   What covers what and how? Please make a clean statement and use 
LaTeX. Here Unicode is not readable enough for me.